### DISCUSSION TITLE
Where do I get the persist plugin?

### DISCUSSION THREAD

**Discussion by PHLAK** [alpinejs/alpine#1489] [1. Help] [answered]
How can I use the `$persist` magic as shown in @calebporzio's Alpine Day talk?

I have Alpine v3 installed and tried the following but get an `Uncaught Error: Cannot find module '@alpinejs/persist'` error.

```js
import persist from '@alpinejs/persist';

Alpine.plugin(persist);
```

```html
<div x-data="{ foo: $persist('bar') }">
   ...
</div>
```

**Comment by benholmen** [comment]
I have the same question, just for `x-trap` :)

**Comment by calebporzio** [comment, marked as answer]
Sorry, the plugins aren't officially available yet, you will have to build them locally and use the built files - or npm link them into your build
